diff -Nurb o.vdk-2.4.1/configure.in n.vdk-2.4.1/configure.in
--- o.vdk-2.4.1/configure.in	2004-11-06 19:16:43.000000000 +0100
+++ n.vdk-2.4.1/configure.in	2005-08-28 02:02:50.092475176 +0200
@@ -2,8 +2,6 @@
 AC_PREREQ(2.52)
 AC_INIT(vdk/vdk.h)
 
-cflags_set=${CFLAGS+set}
-
 VDK_VERSION_MAJOR=2
 VDK_VERSION_MINOR=4
 VDK_VERSION_MICRO=1
@@ -121,28 +119,15 @@
 dnl ==========================
 dnl user configurable options
 dnl ==========================
-AC_ARG_ENABLE(debug,   [  --enable-debug=[no/minimum/yes]  enable debug information [default=no]],,enable_debug=no)
-AC_ARG_ENABLE(opt,     [  --enable-opt=[OPTIMIZATION/no]  Compile with -Ox optimization [default=2]],,enable_opt=2)
+AC_ARG_ENABLE(vdkdebug, [  --enable-vdkdebug=[no/yes]  enable vdkdebug information [default=no]],,enable_debug=no)
 AC_ARG_ENABLE(testvdk, [  --enable-testvdk=[yes/no] Compile testvdk program example [default=yes]],,enable_testvdk=yes)
 dnl doxygen options
 AC_ARG_ENABLE(doc-html,  [ --enable-doc-html[=yes/no] enable generation of docs in html format [default=yes]],,enable_doc_html=yes)
 AC_ARG_ENABLE(doc-latex, [ --enable-doc-latex[=yes/no] enable generation of docs in latex format [default=no]],,enable_doc_latex=no)
 AC_ARG_ENABLE(doc-man,   [ --enable-doc-man[=yes/no] enable generation of docs in latex format [default=no]],,enable_doc_man=no)
 
-if test "x$enable_debug" = "xminimum"; then
-  AC_DEFINE(VDKDEBUG)
-  CXXFLAGS="-s"
-else
-  if test "x$enable_debug" = "xyes"; then
+if test "x$enable_vdkdebug" = "xyes"; then
     AC_DEFINE(VDKDEBUG)
-    CXXFLAGS="-g"
-  else
-    CXXFLAGS="-s"
-  fi
-fi
-
-if test "x$enable_opt" != "xno"; then
-  CXXFLAGS="$CXXFLAGS -O$enable_opt"
 fi
 
 if test "x$enable_testvdk" = "xyes"; then
@@ -179,10 +164,6 @@
 	TAG_SIGC=""
 fi
 
-if test "x$enable_devel" = "xyes"; then
-  cflag="-g"
-fi
-
 if test "x$enable_doc_latex" = "xyes"; then
 	GENERATE_LATEX=YES
 #	TAG_SIGC=USE_SIGCPLUSPLUS
diff -Nurb o.vdk-2.4.1/testvdk/Makefile.am n.vdk-2.4.1/testvdk/Makefile.am
--- o.vdk-2.4.1/testvdk/Makefile.am	2004-11-06 18:47:14.000000000 +0100
+++ n.vdk-2.4.1/testvdk/Makefile.am	2005-08-28 02:45:08.377597184 +0200
@@ -5,7 +5,6 @@
 EXTRA_PROGRAMS = testvdk testvdkstring
 
 INCLUDES = @INCLUDES@
-AM_CXXFLAGS = @CXXFLAGS@
 
 testvdk_LDFLAGS = $(GTK_LIBS)
 testvdk_LDADD = $(top_srcdir)/vdk/libvdk.la
diff -Nurb o.vdk-2.4.1/vdk/Makefile.am n.vdk-2.4.1/vdk/Makefile.am
--- o.vdk-2.4.1/vdk/Makefile.am	2004-11-06 18:47:10.000000000 +0100
+++ n.vdk-2.4.1/vdk/Makefile.am	2005-08-28 02:45:22.032521320 +0200
@@ -186,7 +186,6 @@
 		-version-info $(LT_VDK_CURRENT):$(LT_VDK_REVISION):$(LT_VDK_AGE)
 
 INCLUDES = -I$(top_srcdir)/include @INCLUDES@
-AM_CXXFLAGS = @CXXFLAGS@
 
 AM_LDFLAGS = $(GTK_CFLAGS) 
 LIBS = $(GTK_LIBS)
