diff -burN vdk-2.0.3-oryg/configure.in vdk-2.0.3/configure.in
--- vdk-2.0.3-oryg/configure.in	2003-05-17 06:54:19.000000000 +0200
+++ vdk-2.0.3/configure.in	2004-04-09 22:54:29.000000000 +0200
@@ -174,8 +174,6 @@
 dnl ==========================
 dnl user configurable options
 dnl ==========================
-AC_ARG_ENABLE(debug,   [  --enable-debug=[no/minimum/yes]  enable debug information [default=no]],,enable_debug=no)
-AC_ARG_ENABLE(opt,     [  --enable-opt=[OPTIMIZATION/no]  Compile with -Ox optimization [default=2]],,enable_opt=2)
 AC_ARG_ENABLE(testvdk, [  --enable-testvdk=[yes/no] Compile testvdk program example [default=yes]],,enable_testvdk=yes)
 AC_ARG_ENABLE(sigc, [  --enable-sigc=[yes/no] Enable libsigc++-extension [default=no]],,enable_sigc=no)
 AC_ARG_ENABLE(testsigc, [  --enable-testsigc=[yes/no] Compile testsigcvdk program [default=no]],,enable_testsigc=no)
@@ -194,20 +192,12 @@
 
 if test "x$enable_debug" = "xminimum"; then
   AC_DEFINE(VDKDEBUG)
-  CXXFLAGS="-s"
 else
   if test "x$enable_debug" = "xyes"; then
     AC_DEFINE(VDKDEBUG)
-    CXXFLAGS="-g"
-  else
-    CXXFLAGS="-s"
   fi
 fi
 
-if test "x$enable_opt" != "xno"; then
-  CXXFLAGS="$CXXFLAGS -O$enable_opt"
-fi
-
 if test "x$enable_testvdk" = "xyes"; then
   TESTVDK="testvdk"
 else
@@ -296,14 +286,7 @@
 
 CXXFLAGS="$CXXFLAGS -Wall -Wcast-qual -Woverloaded-virtual -Wconversion -Wstrict-prototypes -Winline"
 
-AC_SUBST(CXXFLAGS)
-
-INCLUDES="$GTK_CFLAGS $SIGC_CFLAGS $SIGCDEFINE"
-
-
-
-AC_SUBST(INCLUDES)
-
+INCLUDES="$GTK_CFLAGS $SIGC_CFLAGS $GNOME_CFLAGS $SIGCDEFINE $GNOMEDEFINE"
 
 
 dnl	  contrib/Makefile 
diff -burN vdk-2.0.3-oryg/sigcvdktest/Makefile.am vdk-2.0.3/sigcvdktest/Makefile.am
--- vdk-2.0.3-oryg/sigcvdktest/Makefile.am	2000-11-28 21:50:17.000000000 +0100
+++ vdk-2.0.3/sigcvdktest/Makefile.am	2004-04-09 22:59:47.000000000 +0200
@@ -1,9 +1,10 @@
-noinst_PROGRAMS= @TESTSIGCVDK@
+noinst_PROGRAMS= $(TESTSIGCVDK)
 
 EXTRA_PROGRAMS = testsigcvdk props_test
 
-INCLUDES = -I$(top_srcdir)/include @INCLUDES@
-CXXFLAGS = @CXXFLAGS@
+INCLUDES = -I$(top_srcdir)/include \
+	   $(GTK_CFLAGS) $(SIGC_CFLAGS) $(GNOME_CFLAGS) \
+	   $(SIGCDEFINE) $(GNOMEDEFINE)
 
 testsigcvdk_LDFLAGS = $(GTK_LIBS) $(SIGC_LIBS)
 testsigcvdk_LDADD = $(top_srcdir)/vdk/libvdk.la 
diff -burN vdk-2.0.3-oryg/testvdk/Makefile.am vdk-2.0.3/testvdk/Makefile.am
--- vdk-2.0.3-oryg/testvdk/Makefile.am	2003-01-23 14:46:23.000000000 +0100
+++ vdk-2.0.3/testvdk/Makefile.am	2004-04-09 22:56:31.000000000 +0200
@@ -1,11 +1,10 @@
 SUBDIRS = .
 
-noinst_PROGRAMS = @TESTVDK@
+noinst_PROGRAMS = $(TESTVDK)
 
 EXTRA_PROGRAMS = testvdk testvdkstring
 
-INCLUDES = @INCLUDES@
-CXXFLAGS = @CXXFLAGS@
+INCLUDES = $(GTK_CFLAGS) $(SIGC_CFLAGS) $(GNOME_CFLAGS) $(SIGCDEFINE) $(GNOMEDEFINE)
 
 testvdk_LDFLAGS = $(GTK_LIBS) $(SIGC_LIBS)
 testvdk_LDADD = $(top_srcdir)/vdk/libvdk.la
@@ -39,10 +38,9 @@
 
 testvdkstring_SOURCES = testvdkstring.cc 
 
-testvdkstring_LDFLAGS = $(GTK_LIBS) $(SIGC_LIBS)
-
-testvdkstring_LDADD = $(top_srcdir)/vdk/libvdk.la
-
+tvdkstring_LDADD = \
+	$(top_srcdir)/vdk/libvdk.la
+	$(GTK_LIBS) $(SIGC_LIBS)
 
 EXTRA_DIST = \
 	loadfile.xpm \
diff -burN vdk-2.0.3-oryg/vdk/Makefile.am vdk-2.0.3/vdk/Makefile.am
--- vdk-2.0.3-oryg/vdk/Makefile.am	2003-05-11 17:08:49.000000000 +0200
+++ vdk-2.0.3/vdk/Makefile.am	2004-04-09 22:58:15.000000000 +0200
@@ -259,8 +259,9 @@
 
 
 
-INCLUDES = -I$(top_srcdir)/include @INCLUDES@
-CXXFLAGS = @CXXFLAGS@
+INCLUDES = -I$(top_srcdir)/include \
+	   $(GTK_CFLAGS) $(SIGC_CFLAGS) $(GNOME_CFLAGS) \
+	   $(SIGCDEFINE) $(GNOMEDEFINE)
 
 LDFLAGS = $(GTK_CFLAGS) $(SIGC_CFLAGS)
 LIBS = $(GTK_LIBS) $(SIGC_LIBS)
